docs/tutorial/automated-testing.md

```diff
@@ -27,29 +27,94 @@ Node.js package for testing with WebDriver. Its ecosystem also includes various
 First you need to run the WebdriverIO starter toolkit in your project root directory:
```

Comment:
I think it would be nice to note in the first paragraph at what WDIO version Electron support was added, in case a project is already using WDIO and needs to upgrade to get the latest capabilities.

Reply:
addressed in https://github.com/electron/electron/pull/40083/commits/a9849acd2604fcefaf2820e5ecee405f37c6bc48